agent/ecscni/plugin_windows.go

```diff
@@ -35,10 +40,32 @@ var (
 		"Amazon", "ECS", "log", "vpc-eni.log")
 )
 
-// setupNS is the called by SetupNS to setup the task namespace by invoking ADD for given CNI configurations
+// setupNS is the called by SetupNS to setup the task namespace by invoking ADD for given CNI configurations.
+// For Windows, we will retry the setup on Windows before conceding error.
```

Comment:
nit: For Windows, we will retry the setup ~~on Windows~~ before conceding error.